sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -413,10 +704,29 @@ def get_partition_properties(self, partition_id):
             partition_id
         )
 
-    def close(self):
-        # type: () -> None
+    def flush(self, **kwargs: Any) -> None:
+        """
+        Buffered mode only.
+        Flush events in the buffer to be sent immediately if the client is working in buffered mode.
+
+        :keyword Optional[float] timeout: Timeout to flush the buffered events, default is None which means no timeout.
+        :rtype: None
```

Comment:
We should also have a description of the error behaviour here, or add a `:raises:` tag.

Reply:
updated!